Source/Resources/Images.xcassets/Schema/Contents.json

```diff
@@ -0,0 +1,6 @@
+{
+  "info" : {
```

Comment:
The folder should be called `Scheme` not `Schema`. They do not mean the same thing and schema has nothing to do with a card scheme.